docs/source/guides/converting-to-openapi.rst

```diff
@@ -114,7 +114,7 @@ The Smithy Gradle plugin relies on a ``smithy-build.json`` file found at the
 root of a project to define the actual process of building the OpenAPI
 specification. The following example defines a ``smithy-build.json`` file
 that builds an OpenAPI specification from a service for the
-``smithy.example#Weather`` service using the ``aws.rest-json-1.1`` protocol:
+``smithy.example#Weather`` service using the ``aws.protocols#restJson1`` protocol:
```

Comment:
We decided to use `restJson1` rather than just `restJson` since we eventually plan to ship an improved REST+JSON protocol.